net/ghttp/ghttp_request_param.go

```diff
@@ -229,6 +229,12 @@ func (r *Request) parseBody() {
 	if r.parsedBody {
 		return
 	}
+
+	// If it's binary data, it does not need to be parsed.
+	if contentType := r.Header.Get("Content-Type"); contentType == "" || gstr.Contains(contentType, "octet-stream") {
```

Comment:
- condition `contentType == ""` might be removed?
- missing associated unit testing case for the changes.